CMakeLists.txt

```diff
@@ -97,3 +99,14 @@ if(NOT ANDROID AND NOT IOS_PLATFORM)
 endif()
 
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/swift/cpp.xcconfig.in ${CMAKE_CURRENT_SOURCE_DIR}/swift/cpp.xcconfig @ONLY)
+
+install(TARGETS TrustWalletCore
+        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+        ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+        )
+
+install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/src/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/WalletCore
```

Comment:
you don't need to use those C++ headers?

Reply:
I use the library in the cpp project, so using C API is less convenient, also overhead when converting from json -> protobuf -> cpp object.

Also, C API does not provide the full functionality of the library, for example, to get a list of selected unspent transactions from the plan, I wanted to create an issue, but I didn't get around.